crates/ruff_linter/src/line_width.rs

```diff
@@ -32,6 +32,12 @@ impl LineLength {
     pub fn text_len(&self) -> TextSize {
         TextSize::from(u32::from(self.value()))
     }
+
+    pub fn from_u16_for_testing_only(length: u16) -> Self {
+        assert!(length >= 1);
+        assert!(length <= 320);
+        Self(length.try_into().unwrap())
+    }
```

Comment:
I'd like to _not_ do this if possible, but I couldn't find a better way to initialize `LineLength`. Maybe I should use `serde_json::from_value(serde_json::Value::Number(80))` instead?